### PR TITLE
[RTC-94] ICE metrics

### PR DESCRIPTION
**Pull request by daniel-jodlos**

### CODE REVIEW

File: lib/membrane_ice_plugin/ice_endpoint.ex [hunk 1]
```diff
@@ -248,6 +253,13 @@ defmodule Membrane.ICE.Endpoint do
           })
           |> start_ice_restart_timer()
 
+        Membrane.TelemetryMetrics.execute(
+          @ice_port_assigned,
+          %{port: candidate_port, protocol: :udp},
```
Comment (FelonEkonom) [inline lib/membrane_ice_plugin/ice_endpoint.ex]: I guess you want to put TURN server port, instead of this

File: mix.exs [hunk 2]
```diff
@@ -44,7 +44,7 @@ defmodule Membrane.ICE.Mixfile do
       {:membrane_funnel_plugin, "~> 0.7.0"},
       {:membrane_telemetry_metrics, "~> 0.1.0"},
       {:bunch, "~> 1.5"},
-      {:fake_turn, "~> 0.3.0"},
+      {:fake_turn, github: "membraneframework/fake_turn", branch: "feature/track-send-errors"},
```
Comment (daniel-jodlos) [inline mix.exs]: Taking a note, we should probably release `fake_turn` and get rid of  github dependency before merging

Comment (FelonEkonom) [reply mix.exs:47]: After merging PR in `fake_turn`, this dependency should be to the `master` branch

File: lib/membrane_ice_plugin/ice_endpoint.ex [hunk 3]
```diff
@@ -287,6 +299,13 @@ defmodule Membrane.ICE.Endpoint do
...
```
Comment (FelonEkonom) [inline lib/membrane_ice_plugin/ice_endpoint.ex]: It should be `udp_integrated_turn.server_port` instead of `candidate_port`

File: mix.exs (hunk 2 above)
Comment (FelonEkonom) [inline mix.exs]: Since PR in `fake_turn` is merged, this dependency should be to the `master` branch

**Review by FelonEkonom — CHANGES_REQUESTED** [review]

**Review by FelonEkonom — APPROVED** [review]

**Pull request merged**